596809: Initscript of the ipmievd service not LSB compliant

Don't print pidfile name to output by default.
And if verbose output is enabled, print the final pidfile name,
not the initial value.

diff -up ipmitool-1.8.11/src/ipmievd.c.orig ipmitool-1.8.11/src/ipmievd.c
--- ipmitool-1.8.11/src/ipmievd.c.orig	2009-02-25 21:38:52.000000000 +0100
+++ ipmitool-1.8.11/src/ipmievd.c	2010-06-02 14:24:41.000000000 +0200
@@ -690,7 +690,6 @@ ipmievd_main(struct ipmi_event_intf * ei
 
 	memset(pidfile, 0, 64);
 	sprintf(pidfile, "%s%d", DEFAULT_PIDFILE, eintf->intf->devnum);
-	lprintf(LOG_NOTICE, "ipmievd: using pidfile %s", pidfile);
 
 	for (i = 0; i < argc; i++) {
 		if (strncasecmp(argv[i], "help", 4) == 0) {
@@ -720,6 +719,7 @@ ipmievd_main(struct ipmi_event_intf * ei
 				__min(strlen((const char *)(argv[i]+8)), 63));
 		}
 	}
+	lprintf(LOG_DEBUG, "ipmievd: using pidfile %s", pidfile);
 
 	/*
 	 * We need to open interface before forking daemon
